services/web-development.html

```diff
@@ -0,0 +1,174 @@
+          {% endfor %}
+        </ul>
+      </div>
+    </div>
+  </div>
+
+  <div class="web_dev_services__web_solutions wrap">
+    <div class="header-2">{{ site.data.web_dev_services.web_solutions.title }}</div>
+    <div class="web_dev_services__list">
+      <div class="header-3">
+        {{ site.data.web_dev_services.web_solutions.subtitle_1 }}
+      </div>
+      <div class="web_dev_services__list-wrap">
+        <img src="/assets/images/web_development_services/branded_cloud.webp" alt="Cloud">
+        <ul class="web_dev_services__list-text list_with_columns">
+          {% for item in site.data.web_dev_services.web_solutions.all %}
+            <li class="web_dev_services__list-item">{{ item.item }}</li>
+          {% endfor %}
+        </ul>
+      </div>
+    </div>
+    <div class="header-2">
+      {{ site.data.web_dev_services.web_solutions.subtitle_2 }}
+    </div>
+  </div>
+
+  <div class="web_dev_services__full_stack wrap">
+    <h1 class="header-3 white-text">
+      {{ site.data.web_dev_services.full_stack.title }}
+    </h1>
+    <div class="description">
+      {{ site.data.web_dev_services.full_stack.description_1 }}
+    </div>
+    <div class="description">
+      {{ site.data.web_dev_services.full_stack.description_2 }}
+    </div>
+    <ul class="web_dev_services__full_stack-list">
+      {% for item in site.data.web_dev_services.full_stack.all %}
+        <li class="web_dev_services__full_stack-list-item inria">
+          <span></span>
+          {{ item.item }}
+        </li>
+      {% endfor %}
+    </ul>
+    <div class="description">
+      {{ site.data.web_dev_services.full_stack.description_3 }}
+    </div>
+  </div>
+
+  <div class="web_dev_services__customized_approach wrap">
+    <h2 class="header-2">
+      {{ site.data.web_dev_services.customized_approach.title }}
+    </h2>
+    <div class="list-with-images">
+      {% for item in site.data.web_dev_services.customized_approach.all %}
+        <div class="list-with-images__item">
+          <div class="list-with-images__item-icon">
+            <img src="/assets/images/web_development_services/{{ item.icon }}" alt="Approach">
+          </div>
+          <div class="list-with-images__item-text">
+            <div class="header-2">{{ item.title }}</div>
+            <div class="description">{{ item.description }}</div>
+          </div>
+        </div>
+      {% endfor %}
+    </div>
+  </div>
+
+  <div class="web_dev_services__technologies wrap">
+    <h2 class="header-2">{{ site.data.web_dev_services.technologies.title }}</h2>
+    <img class="desktop-only" src="/assets/images/web_development_services/technologies_d_table.svg" alt="Technologies">
+    <img class="mobile-only" src="/assets/images/web_development_services/technologies_m_table.svg" alt="Technologies">
+    <a href="/expertise" id="web_dev_technologies" class="button white-text violet-button">
+      {{ site.data.web_dev_services.technologies.button }}
```

Comment:
width for those containers looks incorrect

![image](https://github.com/katatsu12/live_blog/assets/22080708/9386d7ba-7606-4444-a5ff-25dce88ff696)